rust/kcl-lib/tests/execute_engine_error_return/artifact_commands.snap

```diff
@@ -139,59 +139,6 @@ description: Artifact commands execute_engine_error_return.kcl
           "relative": true
         }
       }
-    },
-    {
-      "cmdId": "[uuid]",
-      "range": [],
-      "command": {
-        "type": "enable_sketch_mode",
-        "entity_id": "[uuid]",
-        "ortho": false,
-        "animated": false,
-        "adjust_camera": false,
-        "planar_normal": {
-          "x": 0.0,
-          "y": 0.0,
-          "z": 1.0
-        }
-      }
-    },
-    {
-      "cmdId": "[uuid]",
-      "range": [],
-      "command": {
-        "type": "extrude",
```

Comment:
This demonstrates that the Extrude is never even sent to the engine, because KCL returns an error earlier now.